docs/source/related-work.rst

```diff
@@ -72,14 +72,14 @@ There are a couple of older projects that often get mentioned
 *   Pyro_:  Remote objects / RPC
 
 .. _Luigi: https://luigi.readthedocs.io/en/latest/
```

Comment:
imho it's better to link to the top level rtd URL - as publishers can choose which branch/version to redirect to in their "advanced settings" https://docs.readthedocs.io/en/stable/versions.html#redirects-on-root-urls

```suggestion
.. _Luigi: https://luigi.readthedocs.io/
```

Reply:
eg https://luigi.readthedocs.io goes to en/stable and https://scoop.readthedocs.io/ goes to io/0.7